master/pkg/schemas/expconf/log_pattern_config.go

```diff
@@ -4,31 +4,66 @@ import (
 	"encoding/json"
 	"fmt"
 
-	log "github.com/sirupsen/logrus"
-
 	"github.com/determined-ai/determined/master/pkg/schemas"
+	"github.com/determined-ai/determined/master/pkg/set"
 	"github.com/determined-ai/determined/master/pkg/union"
 )
 
 // LogPoliciesConfigV0 is a list of log policies.
+//
+//go:generate ../gen.sh
 type LogPoliciesConfigV0 []LogPolicyV0
 
+// WithDefaults implements the Defaultable psuedointerface.
+func (b *LogPoliciesConfigV0) WithDefaults() *LogPoliciesConfigV0 {
```

Comment:
```suggestion
func (b LogPoliciesConfigV0) WithDefaults() *LogPoliciesConfigV0 {
```
Don't use a pointer receiver here.

Reply:
For WithDefaults to work, I think you _must_ match the pointerness of the receiver and return value.

So probably want to return a non-pointer as well, I think.

Reply:
Thank you!